connectors/services/access_control_sync_job_execution.py

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+#
+from functools import cached_property
+
+from connectors.protocol import JobStatus, JobType
+from connectors.services.job_execution import JobExecutionService
+
+DEFAULT_MAX_CONCURRENT_ACCESS_CONTROL_SYNCS = 1
+
+
+class AccessControlSyncJobExecutionService(JobExecutionService):
+    name = "sync_access_control"
+    display_name = "access control sync job execution"
+    max_concurrency_config = "service.max_concurrent_access_control_syncs"
+    JOB_TYPES = JobType.ACCESS_CONTROL.value
```

Comment:
Also why is this one full capital, but previous ones are lower case?

Reply:
Good point. It should be a class variable, lower case. Will update.